browserstack/README.md

```diff
@@ -0,0 +1,39 @@
+Selenium Testing in Browserstack
+================================
+
+Defines automated tests to run against the Scots LiDAR website using Selenium.  Selenium tests can be coded in any language but our language of choice is Python.  Runs these automated tests in a defined matrix of OS, OS version, browser and browser version combinations using Browserstack.  The results can be viewed on the Browserstack website.
+
+In an ideal world tests should be atomic, but due to the number of threads we have available with our current licence, minimising running time for pragmatic reasons needs to be prioritised.  For example, the initial JIRA task defined three functional use cases.  Running these across the defined matrix of combinations resulted in approximately 100 individual tests which took 40+ minutes to run.  Combining all of the individual test conditions into a single "functional smoke test" to be run against across the same matrix of combinations halved the total running time, so we moved to this approach.
+
+Current limitations:
+- The test conditions coded so far are examples of what we can do, not complete functional test coverage of the website
+- The tests are currently triggered manually from the command line, rather than running as part of a pipeline
+- The tests are running against an externally visible website - local developer testing has not been investigated
+- Additional visual testing based on snapshots using Percy still needs to be implemented 
+
+# Browserstack account
+
+    You will need a Browserstack account to run the tests - please speak to Jon Parsons
```

Comment:
Let's create a shared dev account to use for this (I'll show you where to put it) and then we can just say here to get the credentials from Keepass

Reply:
done!